src/koa-esm-specifier-transform.ts

```diff
@@ -0,0 +1,57 @@
+import getStream from 'get-stream';
+import isStream from 'is-stream';
+import * as Koa from 'koa';
+import {transformHTMLString} from './transform-html';
+import {transformJavaScriptModuleString} from './transform-javascript-module';
+
+export type TransformSpecifierFunction = (baseURL: string, specifier: string) =>
+    string;
+
+export default middleware;
+export function middleware(transformSpecifier: TransformSpecifierFunction):
```

Comment:
especially for exports, I do `export const middleware = (...) => ...`. Const export bindings are faster for VMs to setup.

Reply:
I converted all the `function x()` to `const x = ()`.